java/server/src/org/openqa/selenium/grid/graphql/Node.java

```diff
@@ -28,12 +30,15 @@
   private final URI uri;
   private final boolean isUp;
   private final int maxSession;
+  private final String capabilities;
 
-  public Node(UUID id, URI uri, boolean isUp , int maxSession) {
+  public Node(UUID id, URI uri, boolean isUp, int maxSession,
+              Map<Capabilities, Integer> capabilities) {
     this.id = Require.nonNull("Node id", id);
     this.uri = Require.nonNull("Node uri", uri);
     this.isUp = isUp;
     this.maxSession = Require.nonNull("Node maxSession", maxSession);
+    this.capabilities = Require.nonNull("Node capabilities", capabilities.toString());
```

Comment:
Better to convert this to a JSON blob rather than the string representation of a java datastructure.